winrt/test.internal/mocks/MockD2DFactory.h

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
 #pragma once
+#pragma warning(disable : 4996)
```

Comment:
Please put this just around the specific functionality that has been deprecated, and use pragma push/pop to restore the warning state immediately after so this warning doesn't end up globally disabled for the whole project.

Reply:
Disabling the warning just for the corresponding method (GetDesktopDpi) did not work, so I disabled the warning for the whole class in the header file.